releasewarrior/commands.py

```diff
@@ -395,6 +395,8 @@ def run(self):
                 release["date"])
             if curr_build.get("graphid"):
                 logger.info("Graph: https://tools.taskcluster.net/push-inspector/#/%s ", curr_build["graphid"])
+            if curr_build.get("graphid_2"):
+                logger.info("Graph 2: https://tools.taskcluster.net/push-inspector/#/%s ", curr_build["graphid_2"])
```

Comment:
right sorry, I'm confusing existing graph2 with the new graphs we are making for tcmigration relpro